### PR TITLE
Add TrackJSON Download and SkipTranscoding Options.

### PR DESCRIPTION
**Pull request by BryonLewis**
- Annotation download button modified within web version to have a dropdown to select the TrackJSON instead of always exporting the viame-csv format
- Logic included preventing transcoding when uploaded with the `skipTranscoding` parameter set to true.  Should only be done by those who know that their video is h264 encoded.

### CODE REVIEW

**Comment by BryonLewis** [comment]
> > * Logic included preventing transcoding when uploaded with the `skipTranscoding` parameter set to true.  Should only be done by those who know that their video is h264 encoded.
> 
> What happens if you knowingly enable this option on a video that's _not_ in the right format? (i.e., what kind of error message or error state will the user find themselves in?)

Currently, it will fall back on transcoding the video.  That is probably not desired when importing videos and having that option set.  I can easily change it to create an error in the job log.

**Comment by BryonLewis** [comment]
Note to self:  Clarify the setting as being "Attempt to Skip transcoding".  It will fall back to transcoding if the video files are not in the proper format.  Log this information in the Job log and update the UI elements to accurately indicate to the user this setting (may need to use a help icon tip for the longer explanation).

File: server/dive_tasks/tasks.py [hunk 1]
```diff
@@ -442,6 +444,36 @@ def convert_video(self: Task, folderId: str, itemId: str, user_id: str, user_log
...
+                    constants.FPSMarker: newAnnotationFps,
+                    "ffprobe_info": videostream[0],
+                },
+            )
+            return
+        elif skip_transcoding:
+            print('Transcoding can not be skipped:')
+            print(f'Codec Name: {videostream[0]["codec_name"]}')
+            print('If Codec name is not h264 so file will be transcoded')
+
```
Comment (marySalvi) [inline server/dive_tasks/tasks.py:476]:
```suggestion
            print('Transcoding cannot be skipped:')
            print(f'Codec Name: {videostream[0]["codec_name"]}')
            print('Codec name is not h264 so file will be transcoded')

```

File: client/platform/web-girder/views/Upload.vue [hunk 2]
```diff
@@ -505,6 +506,29 @@ export default defineComponent({
...
+                <template #activator="{ on }">
+                  <v-icon
+                    small
+                    v-on="on"
+                  >
+                    mdi-help
+                  </v-icon>
+                </template>
+                <span>Attempt to skip transcoding of video file if it is an
+                  '.mp4' and encoding using the h264 codec.
```
Comment (marySalvi) [inline client/platform/web-girder/views/Upload.vue:528]:
```suggestion
                  '.mp4' and encoded using the h264 codec.
```

**Review by marySalvi — CHANGES_REQUESTED** [review]
Looks great, working well. Just minor grammar changes.

**Review by marySalvi — APPROVED** [review]

**Review by marySalvi — APPROVED** [review]

**Pull request merged**